### PR TITLE
ClimKern Notebook

### PR DESCRIPTION
**Pull request by tyfolino**
Created a notebook for Section 3 of the book and quickly made a surface albedo feedback picture. I don't expect this to pass testing until I update how to get the kernel data.

### CODE REVIEW

**Comment by tyfolino** [comment]
Ok, don't worry, I fixed the kernel issue that was making it fail :).

File: notebooks/simplified-calc/climkern-calc.ipynb [hunk 1]
```diff
@@ -78,41 +60,1365 @@
...
+    "# Save path for later\n",
+    "path_out = files[0].split(kernel+\"/\",1)[1]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To save this data in ClimKern's directory, we have to figure out where it is on the\n",
+    "machine. After that, we will go ahead anad save out the kernel as a netCDF in \n",
```
Comment (anacmontoya) [inline notebooks/simplified-calc/climkern-calc.ipynb:153]:
```suggestion
    "machine. After that, we will go ahead and save out the kernel as a netCDF in \n",
```

File: notebooks/simplified-calc/climkern-calc.ipynb [hunk 2]
```diff
@@ -78,41 +60,1365 @@
...
+    "climkern's local data directory."
```
Comment (anacmontoya) [inline notebooks/simplified-calc/climkern-calc.ipynb:154]:
```suggestion
    "ClimKern's local data directory."
```

**Review by anacmontoya — COMMENTED** [review]
Left a couple of comments, just typos

File: notebooks/simplified-calc/climkern-calc.ipynb (hunk 1 above)
Comment (tyfolino) [reply notebooks/simplified-calc/climkern-calc.ipynb:158]: Thank you for catching these!

**Pull request merged**